packages/core/src/tegel-light/components/tl-dropdown/tl-dropdown.scss

```
@@ -0,0 +1,543 @@
+@import './tl-dropdown-vars';
+
```

I think the import for the typography is missing here
```suggestion
@import '../../../../../../typography/utilities/typography-utility';
```